tools/sil-opt/SILOpt.cpp

```diff
@@ -116,11 +116,11 @@ EnableSpeculativeDevirtualization("enable-spec-devirt",
                   llvm::cl::desc("Enable Speculative Devirtualization pass."));
 
 namespace {
-enum EnforceExclusivityMode {
+enum class EnforceExclusivityMode {
```

Comment:
I made this change since EnforceExclusivityMode was injecting None into the global namespace meaning the enum field clashed with Optional::None. We already were always referring to these enum values explicitly, so it makes sense to just an enum class here.